cdap-app-fabric/src/main/java/co/cask/cdap/app/guice/InMemoryProgramRunnerModule.java

```diff
@@ -111,6 +114,8 @@ protected void configure() {
     // Create webapp http handler factory.
     install(new FactoryModuleBuilder().implement(JarHttpHandler.class, IntactJarHttpHandler.class)
               .build(WebappHttpHandlerFactory.class));
+
+    bind(Store.class).to(DefaultStore.class).in(Scopes.SINGLETON);
```

Comment:
And why program runner need `Store`? The design of the whole `ProgramRunner` and related class is that they should never communicate to Store.